src/entry-editable.ts

```diff
@@ -8,27 +8,91 @@ export function addTags(entry: EntryModel, contentTypeUid: string, tagsAsObject:
 function getTag(content: object, prefix: string, tagsAsObject: boolean, locale: string): object {
     const tags: any = {}
     Object.entries(content).forEach(([key, value]) => {      
+        if (key === '$') return
+
         switch (typeof value) {
             case "object":
                 if (Array.isArray(value)) {
                     value.forEach((obj, index) => {
+                        const childKey = `${key}__${index}`
+                        const parentKey = `${key}__parent`
```

Comment:
Why this comment is under the code ?

Reply:
These are Deepak's changes @ishaileshmishra , I added them as is.

Reply:
I'll remove and recommit. @ishaileshmishra